(Note that the spaces before the blank context lines are missing.)

Index: README
===================================================================
--- README	(revision 3)
+++ README	(working copy)
@@ -1,5 +1,5 @@
 The secret word needed to infiltrate the compound is in word.txt. You
-also will need to be wearing a red hat.  Good luck on your mision!
+also will need to be wearing a red hat.  Good luck on your mission!

   -- Mr. Bad

